tbd.schema.json

```diff
@@ -174,6 +198,18 @@
           "title": "Convex floor/balcony edge PSI ",
           "type": "number"
         },
+        "balconysill": {
+          "title": "Floor/balcony/sill edge PSI ",
+          "type": "number"
+        },
+        "balconysillconcave": {
+          "title": "Concave floor/balcony/sill edge PSI ",
+          "type": "number"
+        },
+        "balconysillconvex": {
+          "title": "Convex floor/balcony/sill edge PSI ",
+          "type": "number"
+        },
```

Comment:
New admissible edge types:

- **roofs** (instead of strictly **parapets**)
- **spandrel** panel edges
- **balconysill** edges

... as well as concave and convex variants of each.